API/TreeData/TreeDataMock.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+using MySql.Data;
+using MySql.Data.MySqlClient;
+
+namespace TreeData_CLI
+{
+    public class TreeRequest

```

Comment:
Since this isn't a request, this is just a tree, then the name for this class should be `Tree`. To keep things organized, `Tree` should be in a file `WebApi/Model/Contracts/Tree.cs`

Reply:
Fixed in https://github.com/ForestGeoHack/ForestGEO/pull/12/commits/0543202ea320047257a303f345157882127330c9

Reply:
It looks like the class was renamed but the `Tree.cs` is not in the right folder - `WebApi/Model/Contracts/`. Please update the namespace accordingly when moving the file.

Reply:
Moved around in https://github.com/ForestGeoHack/ForestGEO/pull/12/commits/08b0ea574748bcf24973780a707cf1f794a598c5